packages/wallets/src/features/cashier/modules/WithdrawalFiat/WithdrawalFiat.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { useWithdrawalFiatAddress } from '@deriv/api';
+import { Loader } from '../../../../components';
+import { WithdrawalVerificationModule } from '../WithdrawalVerification';
+import './WithdrawalFiat.scss';
+
+const WithdrawalFiat = () => {
+    const verification_code = sessionStorage.getItem('verification_code');
+    const { data: iframeUrl, mutate } = useWithdrawalFiatAddress(verification_code!);
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
Why cant we use isLoading from the hook ?

Reply:
Updated